.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "\U0001F41B Bug Report"
+labels: "bug"
```

Comment:
```suggestion
```

currently we don't care about issue type and have no labels for them.

Reply:
addressed